jadx-core/src/main/java/jadx/core/xmlgen/ManifestAttributes.java

```diff
@@ -135,7 +135,7 @@ private void parseValues(String name, NodeList nodeList) {
 					if (attr == null) {
 						return;
 					}
-					attrMap.put(name, attr);
+					attrMap.put(String.format("android:%s", name), attr);
```

Comment:
For performance reasons I would prefer simple string concat here `"android:" + name´. 
For Maps with `String` as key as `attrMap` I am a fan of having a short JavaDoc where the Map is defined where the String key format is described. Could you please add such a JavaDoc comment?